web/src/components/home/subcomponents/upcoming_events/upcoming_events.js

```diff
@@ -1,5 +1,92 @@
+import React from "react";
+import Box from "@mui/material/Box";
+import Button from "@mui/material/Button";
+import Container from "@mui/material/Container";
+import DyosRenderer from "./event_renderers/dyos_renderer";
+import KeyboardArrowLeft from "@mui/icons-material/KeyboardArrowLeft";
+import KeyboardArrowRight from "@mui/icons-material/KeyboardArrowRight";
+import MobileStepper from "@mui/material/MobileStepper";
+import Typography from "@mui/material/Typography";
```

Comment:
These imports can be organized (all `@mui` imports together) and simplified since they're probably named exports:

```suggestion
import { Box, Button, Container, MobileStepper, Typography } from "@mui/material";
import { KeyboardArrowLeft, KeyboardArrowRight } from "@mui/icons-material";
import DyosRenderer from "./event_renderers/dyos_renderer";
```